src/transformStateWithClones.js

```diff
@@ -7,7 +7,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const clonesArray = [];
```

Comment:
don't use data type names in variable names
```suggestion
  const states = [];
```